exp/controllers/azuremanagedcluster_controller.go

```diff
@@ -100,12 +98,7 @@ func (r *AzureManagedClusterReconciler) Reconcile(ctx context.Context, req ctrl.
 	defer cancel()
 	log := r.Log.WithValues("namespace", req.Namespace, "azureManagedCluster", req.Name)
 
-	ctx, span := tele.Tracer().Start(ctx, "controllers.AzureManagedClusterReconciler.Reconcile",
```

Comment:
Again, the attributes are valuable. Please consider reverting the change to Start.

Reply:
sorry about that! re-added in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1460/commits/f21e1c8a0c67c16b223d2d3e3ac57d00706a36a1